README.md

```diff
@@ -473,87 +388,104 @@ The `s` field MUST contain a [Varsig] of the [CBOR] encoded `scope` field.
 
-## 5.2 Fields
+If `meta` field is not present, it is implicitly a `unit` represented as an empty map.
 
-### 5.2.1 UCAN Task Version
+### 5.1.3 Proofs
 
-The `v` field MUST contain the SemVer-formatted version of the UCAN Invocation Specification that this struct conforms to.
+The `prf` field MUST contain links to any UCANs that provide the authority to perform this task. All of the outermost proofs MUST have `aud` field set to the [Executor]'s DID. All of the outmost proofs MUST have `iss` field set to the [Invoker]'s DID.
 
-### 5.2.2 Task
+### 5.1.4 Optional Cause
 
-The `run` field MUST contain a link to the [Task] to be run.
+[Task]s MAY be invoked as an effect caused by a prior [Invocation]. Such [Invocation]s SHOULD have a `cause` field set to the [Receipt] link of the [Invocation] that caused it. The linked [Receipt] MUST have an `Effect` (the `fx` field) containing invoked [Task] in the `run` field.
 
-### 5.2.3 Cause
+## 5.2 (Signed) Invocation
 
-[Task]s MAY be invoked as an effect caused by a prior [Invocation]. Such [Invocation]s SHOULD have a `cause` field set to the [Receipt] link of the [Invocation] that caused it. The linked [Receipt] MUST have an `Effect` (the `fx` field) containing invoked [Task] in the `run` field.
+An [Invocation] is a signed [Task].
+
+```
+type Invocation struct {
+  task     Task
+  auth     &Authorization
+}
+```
 
-### 5.2.4 Authorization
+### 5.2.1 Task
 
-The `auth` field MUST contain a link to the [Authorization] that authorizes invoked [Task] in the `run` field. The linked [Authorization] MUST contain `run` in its `scope`.
+The [Task] containing the [Instruction] and any configuration.
 
-### 5.2.4 Proofs
+### 5.2.2 Authorization
 
-The `prf` field MUST contain links to any UCANs that provide the authority to perform this task. All of the outermost proofs MUST have `aud` field set to the [Executor]'s DID. All of the outmost proofs MUST have `iss` field set to the [Invoker]'s DID.
+The `auth` field MUST contain a [Varsig] signing over an array of CIDs that includes the [Task]'s CID, signed by the issuer of the proofs.
 
-### 5.2.6 Metadata
+## 5.3 Invocation Tag
 
-The OPTIONAL `meta` field MAY be used to include human-readable descriptions, tags, execution hints, resource limits, and so on. If present, the `meta` field MUST contain a map with string keys. The contents of the map are left undefined to encourage extensible use.
+An invocation capsule associates the [Invocation] with a versioned schema. This field is NOT REQUIRED. Wrapping an [Invocation] in an Invocation Tag is RECOMMENDED in contexts where the schema and version are not clear from context, such as when it is being stores or passed around without being nested in another structure that defines the version in its schema.
 
-If `meta` field is not present, it is implicitly a `unit` represented as an empty map.
+```
+type InvocationTag union {
```

Comment:
Atm Tags hang loose, outside the DAG. E.g., Reciept references Invocation and not Invocation Tag, leaving the Tag out. I.e.,
```
Receipt ------> Invocation -> Task -> Instruction
Invocation Tag -^
```
Same with Reciept Tag.
Perhaps we could extend any spec that may contain Tagged types with such support, e.g.,
```
type Receipt struct {
  ran     &Invocation | &InvocationTag
}
```

That seems to comlicate the spec and add another layer of wrapping (and CID resolution) (resolve Receipt, *resolve InvocationTag*, resolve Invocation), as well as making Invocation's context-dependent / not self-contained / not having sufficient description on their own.

Perhaps another way is to extend Invocation itself with an optional `version` field?
Akin to how it's done for UCANs [with `ucv`](https://github.com/ucan-wg/spec#321-version)?

Reply:
Yeah, this is all getting moved out of this spec and over to the IPLD representation (which is its own spec now). "Capsule types" are a pattern in that ecosystem, but we don't want it to spread through every possible representation.